packages/rest/test/acceptance/bootstrapping/application.acceptance.ts

```diff
@@ -5,35 +5,40 @@
 
 import {expect, supertest} from '@loopback/testlab';
 import {Constructor, Provider, inject} from '@loopback/context';
-import {Application, DefaultSequence, CoreBindings} from '../../..';
+import { DefaultSequence, RestBindings, RestServer } from '../../..';
+import {Application} from '@loopback/core';
 
 describe('Bootstrapping the application', () => {
   context('with a user-defined sequence', () => {
     let app: Application;
+    let server: RestServer;
     before(givenAppWithUserDefinedSequence);
 
     it('binds the `sequence` key to the user-defined sequence', async () => {
-      const binding = await app.get(CoreBindings.SEQUENCE);
+      const binding = await server.get(RestBindings.SEQUENCE);
       expect(binding.constructor.name).to.equal('UserDefinedSequence');
     });
 
     function givenAppWithUserDefinedSequence() {
       class UserDefinedSequence extends DefaultSequence {}
-      app = new Application({
+      app = new Application();
+      server = new RestServer(app, {
         sequence: UserDefinedSequence,
       });
+      app.server(server);
     }
   });
 
   context('with user-defined components', () => {
     it('binds all user-defined components to the application context', () => {
       class AuditComponent {}
-
       const app = new Application({components: [AuditComponent]});
-      const componentKeys = app.find('component.*').map(b => b.key);
+      const server = new RestServer(app);
+      app.server(server);
+      const componentKeys = server.find('component.*').map(b => b.key);
       expect(componentKeys).to.containEql('components.AuditComponent');
 
-      const componentInstance = app.getSync('components.AuditComponent');
+      const componentInstance = server.getSync('components.AuditComponent');
```

Comment:
ditto: I think this change should be reverted - components are bound at application level now, right?